src/v2/Apps/Fair/Components/FairExhibitorRail/FairExhibitorRail.tsx

```diff
@@ -19,21 +19,13 @@ interface FairExhibitorRailProps extends BoxProps {
   show: FairExhibitorRail_show
 }
 
-/**
- * Though it is likely to exist, the sale message line may be missing.
- * In order to avoid the page shifting between the loading state and the ready state,
- * we need to hardcode the height.
- */
-export const FAIR_EXHIBITOR_RAIL_HEIGHT = 233
-export const FAIR_EXHIBITOR_IMAGE_HEIGHT = 160
-
 export const FairExhibitorRail: React.FC<FairExhibitorRailProps> = ({
   show,
   ...rest
 }) => {
   const ref = useRef<HTMLDivElement | null>(null)
   const tracking = useTracking()
-  const { isEnteredView, Waypoint } = useLazyLoadComponent()
+  const { isEnteredView, Waypoint } = useLazyLoadComponent({ threshold: 2000 })
```

Comment:
I hate to say I don't think I ever enabled batching in production. Which might give us a nice performance boost.